themes/reference/overriding-modules.md

```diff
@@ -97,7 +98,7 @@ The issue goes both ways: if you want to modify the `included-template.tpl` file
 {include file='./included-template.tpl'}
 ```
 
-PrestaShop 1.7 introduces a new cool way to include files in module templates. By using this method, all the expected rules will be followed:
+PrestaShop 1.7 introduces a new way to include files in module templates. By using this method, all the expected rules will be followed:
```

Comment:
"Cool" is not something I personally expect from a documentation for developers.